http://bugs.gentoo.org/155125
http://lists.gnu.org/archive/html/bug-coreutils/2006-11/msg00218.html

2006-11-26  Mike Frysinger  <vapier@gentoo.org>

	* m4/acl.m4 (AC_FUNC_ACL): Wrap all ACL logic in a call to
	AC_ARG_ENABLE(acl).

--- m4/acl.m4
+++ m4/acl.m4
@@ -12,6 +12,11 @@
   AC_LIBOBJ([acl])
   AC_LIBOBJ([file-has-acl])
 
+AC_ARG_ENABLE(acl,
+  [  --disable-acl           turn off support for ACLs],
+  use_acl=$enableval, use_acl=yes)
+
+if test "$use_acl" = "yes"; then
   dnl Prerequisites of lib/acl.c.
   AC_CHECK_HEADERS(sys/acl.h)
   AC_CHECK_FUNCS(acl)
@@ -39,6 +44,9 @@
       AC_REPLACE_FUNCS([acl_entries])
     fi
   LIBS="$ac_save_LIBS"
+else
+  use_acl=0
+fi
   if test $use_acl = 1; then
     ac_save_LIBS="$LIBS"
     AC_SEARCH_LIBS([acl_trivial], [sec],
